sdk/servicebus/service-bus/package.json

```diff
@@ -99,7 +99,7 @@
     "@azure/abort-controller": "^1.0.0",
     "@azure/core-amqp": "2.0.0-beta.1",
     "@azure/core-asynciterator-polyfill": "^1.0.0",
-    "@azure/core-http": "^1.2.0",
+    "@azure/core-http": "^1.1.6",
```

Comment:
could we use the latest released 1.1.9?

Reply:
Well, 1.1.6 was the last used version, and hence picked that.
I don't see a reason to not update to 1.1.9.  Will do.